Pod/Classes/GDGCondition.m

```diff
@@ -93,7 +93,7 @@ - (instancetype)init
 		};
 
 		_equalsCol = ^GDGCondition *(GDGColumn *column) {
```

Comment:
This block bothers me. 

A better API would just have the block `^equals` and would handle if the parameter is a `GDGColumn` which is, I should point, a known parameter possibility